update/images.go

```diff
@@ -38,32 +38,26 @@ func (r ImageRepos) GetRepoImages(repo image.Name) ImageInfos {
 // ImageInfos is a list of image.Info which can be filtered.
 type ImageInfos []image.Info
 
-// Filter returns only the images which match the tagGlob.
-func (ii ImageInfos) Filter(tagGlob string) ImageInfos {
-	var filtered ImageInfos
-	for _, i := range ii {
-		tag := i.ID.Tag
-		// Ignore latest if and only if it's not what the user wants.
-		if !strings.EqualFold(tagGlob, "latest") && strings.EqualFold(tag, "latest") {
-			continue
-		}
-		if glob.Glob(tagGlob, tag) {
-			var im image.Info
-			im = i
-			filtered = append(filtered, im)
-		}
-	}
-	return filtered
+// SortedImageInfos is a list of sorted image.Info
+type SortedImageInfos []image.Info
```

Comment:
(and below ..) Oh cool, neat job. Could you make the fields of api/v6/Container `SortedInfos`, or does that screw things up?